hystrix-contrib/hystrix-metrics-event-stream/src/main/java/com/netflix/hystrix/contrib/metrics/eventstream/HystrixMetricsPoller.java

```diff
@@ -134,19 +134,6 @@ public boolean isRunning() {
     /**
      * Used to protect against leaking ExecutorServices and threads if this class is abandoned for GC without shutting down.
      */
-    @SuppressWarnings("unused")
```

Comment:
This one does get called, just not within the Hystrix library.  See http://stackoverflow.com/questions/6872857/how-does-a-finalizer-guardian-work-in-java for a full explanation of how this works.  This one should stay.